airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.kt

```diff
@@ -28,6 +29,12 @@ class AirbyteExceptionHandler : Thread.UncaughtExceptionHandler {
         // https://docs.google.com/document/d/1ctrj3Yh_GjtQ93aND-WH3ocqGxsmxyC3jfiarrF6NY0/edit#
         LOGGER.error(logMessage, throwable)
 
+        val rootThrowable = ConnectorExceptionUtil.getRootConfigError(Exception(throwable))
+
+        if (ConnectorExceptionUtil.isConfigError(rootThrowable)) {
```

Comment:
Since destinations also use this handler. Is the expected behavior shared by them as well? 
cc: @jbfbell

Reply:
This makes sense to me, so essentially if it's a config error, don't throw two errors?

Reply:
exactly - it will still exit 1, but we are not sending a system_error trace message anymore to platform if it's a config error.